pkg/webhooks/server.go

```diff
@@ -112,45 +104,45 @@ func (ws *WebhookServer) RunAsync() {
 	go func(ws *WebhookServer) {
 		err := ws.server.ListenAndServeTLS("", "")
 		if err != nil {
-			ws.logger.Fatal(err)
+			glog.Fatal(err)
 		}
 	}(ws)
-
-	ws.logger.Printf("Started Webhook Server")
+	glog.Info("Started Webhook Server")
 }
 
 // Stop TLS server and returns control after the server is shut down
 func (ws *WebhookServer) Stop() {
 	err := ws.server.Shutdown(context.Background())
 	if err != nil {
 		// Error from closing listeners, or context timeout:
-		ws.logger.Printf("Server Shutdown error: %v", err)
+		glog.Info("Server Shutdown error: %v", err)
 		ws.server.Close()
 	}
 }
 
 // HandleMutation handles mutating webhook admission request
 func (ws *WebhookServer) HandleMutation(request *v1beta1.AdmissionRequest) *v1beta1.AdmissionResponse {
-	ws.logger.Printf("Handling mutation for Kind=%s, Namespace=%s Name=%s UID=%s patchOperation=%s",
+	glog.Infof("Handling mutation for Kind=%s, Namespace=%s Name=%s UID=%s patchOperation=%s",
```

Comment:
According to https://github.com/nirmata/kyverno/issues/41#issuecomment-497371716 add logging of raw object to this handler and validation handler:
1. Log basic resource information to v(LogInfo)
2. Log raw resource to v(LogTrace)